modules/crud/sql/read.go

```diff
@@ -101,6 +113,27 @@ func (s *SQL) Read(ctx context.Context, project, col string, req *model.ReadRequ
 	}
 
 	switch req.Operation {
+	case utils.Count:
+		mapping := make(map[string]interface{})
+		if !rows.Next() {
+			return nil, errors.New("SQL: No response from db")
+		}
+
+		err := rows.MapScan(mapping)
+		if err != nil {
+			return nil, err
+		}
+
+		switch s.GetDBType() {
+		case utils.MySQL, utils.Postgres:
+			mysqlTypeCheck(rowTypes, mapping)
+		}
+
+		for _, v := range mapping {
+			return v, nil
```

Comment:
What are we exactly returning here? The count number? I think we should return an object with key count. Whats the mongo api returning?

Reply:
Yes, we are returning a number. The mongo api also returns a count number.